### PR TITLE
changes requested by godaddy

### PR DESCRIPTION
**Pull request by Thekkedath**

### CODE REVIEW

File: goentri.com.clickfunnels.json [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
+  "description": "Allows user to easily set up domain using Entri",
+  "syncPubKeyDomain": "goentri.com",
+  "syncRedirectDomain": "api.goentri.com, goentri.com, entri.com",
+  "variableDescription": "verificationValue is the value that is used to verify domain ownership,clusterValue is the cluster assigned to the domain,websiteId is the id of website, keyValue is the generated key value",
+  "sharedProviderName": true,
+  "hostRequired": true,
+  "records": [
+    {
+      "type": "TXT",
+      "host": "@",
```
Comment (pawel-kow) [inline goentri.com.clickfunnels.json:17]: Hey, I really don't get this change now and what you want to achieve. I feel like running in circles :)

May you please give 1 full example of variables you would send and what DNS records (with full host names) you expect to see in the zone?

**Review by pawel-kow — CHANGES_REQUESTED** [review]

File: goentri.com.clickfunnels.json (hunk 1 above)
Comment (Thekkedath) [reply goentri.com.clickfunnels.json:17]: Sure. This is the sample set of DNS records that clickfunnel will be sending
>[
      {
        "value": "us-east-1.myclickfunnels.com",
        "host": "www",
        "ttl": 300,
        "type": "CNAME"
      },
      {
        "value": "clickfunnels-domain-verification=vDbwbJ",
        "host": "@",
        "ttl": 300,
        "type": "TXT"
      },
      {
        "value": "k=rsa; p=MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAoo1QvVJsE6pwt3D5I0qyUaLm3fZeoVpIf3kQj4UoaOAd6z06hkCgRaYjdHQyNmiKPSxm8ro8p2aJdm7aOHnBATe27ppSWOHIssGMUz1ip4UCAHplpFC3hdMDEb4K4SS8F1gQy+55U5GKvRilm4D+YTFWpYBG3mh06JWLFdWtZpq1ukA09SpWQxfiycKAZlrNyxTqmWgHbEb6/uYbxIEsifiu9afYJF1J6RiatrI3M1eGznriogkweMJyh15TRX72Gdxa2cFDwl+6tqtfdZP8M/kAAdeG3PsA1ZJPBZrwv9001EXq+vkf4woLVnNmSGg04aRFdXqO4GUNaM3qCwicaQIDAQAB",
        "host": "1651286188._domainkey.www",
        "ttl": 300,
        "type": "TXT"
      }
    ]
    
And were planning to use the DC URL in the following format for this sample recordset.

`https://dcc.godaddy.com/manage/v2/domainTemplates/providers/goentri.com/services/clickfunnel/apply?domain=example.com&verificationValue=vDbwbJ&clusterValue=us-east-1.myclickfunnels.com&websiteId=1651286188&keyValue=k=rsa; p=MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAoo1QvVJsE6pwt3D5I0qyUaLm3fZeoVpIf3kQj4UoaOAd6z06hkCgRaYjdHQyNmiKPSxm8ro8p2aJdm7aOHnBATe27ppSWOHIssGMUz1ip4UCAHplpFC3hdMDEb4K4SS8F1gQy+55U5GKvRilm4D+YTFWpYBG3mh06JWLFdWtZpq1ukA09SpWQxfiycKAZlrNyxTqmWgHbEb6/uYbxIEsifiu9afYJF1J6RiatrI3M1eGznriogkweMJyh15TRX72Gdxa2cFDwl+6tqtfdZP8M/kAAdeG3PsA1ZJPBZrwv9001EXq+vkf4woLVnNmSGg04aRFdXqO4GUNaM3qCwicaQIDAQAB&providerName=Clickfunnel`

Comment (pawel-kow) [reply goentri.com.clickfunnels.json:17]: You can't use the link like this, you need to add host=www.
If you set it the result won't be as you've expected:

```
{
  "records": [
    {
      "type": "TXT",
      "fqdn": "www.example.com.",
      "data": "clickfunnels-domain-verification=vDbwbJ",
      "ttl": 3600
    },
    {
      "type": "CNAME",
      "fqdn": "www.example.com.",
      "pointsTo": "us-east-1.myclickfunnels.com",
      "ttl": 3600
    },
    {
      "type": "TXT",
      "fqdn": "1651286188._domainkey.www.example.com.",
      "data": "k=rsa; p=MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAoo1QvVJsE6pwt3D5I0qyUaLm3fZeoVpIf3kQj4UoaOAd6z06hkCgRaYjdHQyNmiKPSxm8ro8p2aJdm7aOHnBATe27ppSWOHIssGMUz1ip4UCAHplpFC3hdMDEb4K4SS8F1gQy+55U5GKvRilm4D+YTFWpYBG3mh06JWLFdWtZpq1ukA09SpWQxfiycKAZlrNyxTqmWgHbEb6/uYbxIEsifiu9afYJF1J6RiatrI3M1eGznriogkweMJyh15TRX72Gdxa2cFDwl+6tqtfdZP8M/kAAdeG3PsA1ZJPBZrwv9001EXq+vkf4woLVnNmSGg04aRFdXqO4GUNaM3qCwicaQIDAQAB",
      "ttl": 3600
    }
  ]
}
```

The CNAME needs either `%domain%.` to point to the the apex `example.com.` or any other variable, say `%domainapex%.`, if DG does not support `%domain%` as per spec 6.9.2, which you will pass with the same value as `domain` parameter.

Comment (Thekkedath) [reply goentri.com.clickfunnels.json:17]: Sorry I skipped adding the host=www (and also signature param). Using `%domainapex%` for CNAME record makes sense but was wondering if I should use the same for TXT record or just keep it as `@`?

Comment (pawel-kow) [reply goentri.com.clickfunnels.json:17]: Oh my bad, my last comment was about the TXT record which is to land at APEX, not the CNAME record. The comment corrected above. CNAME shall remain on `@`, TXT with `%domainapex%.`. The trailing dot is really important here.

Comment (Thekkedath) [reply goentri.com.clickfunnels.json:17]: no problem. Will make that change now. Thanks again for helping us with this.

**Review by pawel-kow — APPROVED** [review]

**Review by bdickson1-godaddy — APPROVED** [review]
Looks fine.

**Pull request merged**